### PR TITLE
Follow-up edits to PR#5005

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/5005

Preview Build: http://file.rdu.redhat.com/~ahardin/08282017/PR5005-followup/admin_guide/encrypting_data.html

### CODE REVIEW

File: admin_guide/encrypting_data.adoc [hunk 1]
```diff
@@ -286,5 +298,10 @@ resources:
           secret: <BASE 64 ENCODED SECRET>
 ----
 
-and restart all master processes. Then run the command `oc adm migrate storage --include=secrets --confirm`
-to force all secrets to be decrypted.
+. Restart all master processes.
```
Comment (ahardin-rh) [inline admin_guide/encrypting_data.adoc]: @php-coder Is this the correct command for this step? I think we should show it:
# systemctl restart atomic-openshift-master

File: admin_guide/encrypting_data.adoc [hunk 2]
```diff
@@ -159,58 +158,58 @@ resources:
...
-Python are not suitable.
+The encryption key must be generated with an appropriate cryptographically
+secure random number generator like *_/dev/urandom_*. For example, `math/random`
+from Golang or `random.random()` from Python are not suitable.
 ====
 
-. Place that value in the *secret* field
+. Place that value in the `secret` field.
 
-. Restart API server
```
Comment (ahardin-rh) [inline admin_guide/encrypting_data.adoc]: @php-coder What must be run to restart the API server?

File: admin_guide/encrypting_data.adoc (hunk 1 above)
Comment (php-coder) [reply admin_guide/encrypting_data.adoc]: It turned out that restarting API server(s) should be enough in this case:
```
<semushin> claytonc, in the doc about encryption at rest once you suggest to restart "all master processes" and in another place "restart API server". Why do we need to restart a full master? Sound like restarting API server should be sufficient in all the cases.
<claytonc> semushin: yes apiserver is minimal
<claytonc> that's what should be restarted
<claytonc> for secrets
```

File: admin_guide/encrypting_data.adoc (hunk 2 above)
Comment (php-coder) [reply admin_guide/encrypting_data.adoc]: Based on our documentation, looks like the commands are different and depend on the setup:
1) for OpenShift Origin the command is `systemctl restart origin-master`
2) for OpenShift Enterprise on a single master cluster installation the command is `systemctl restart atomic-openshift-master`
3) for OpenShift Enterprise on a multi-master cluster installation the command will be `systemctl restart atomic-openshift-master-api` (on each master)

Here is the possible asciidoc variant:
```
ifdef::openshift-origin[]
----
# systemctl restart origin-master
----
endif::[]
ifdef::openshift-enterprise[]
On a single master cluster installation:
----
# systemctl restart atomic-openshift-master
----
On a multi-master cluster installation, on each master:
----
# systemctl restart atomic-openshift-master-api
----
endif::[]
```

@smarterclayton Could you confirm that these commands are correct?

File: admin_guide/encrypting_data.adoc (hunk 1 above)
Comment (ahardin-rh) [reply admin_guide/encrypting_data.adoc]: @php-coder So remove all mentions of restarting master processes across the page, or just this section?

Comment (php-coder) [reply admin_guide/encrypting_data.adoc]: Not removing but replacing:
```diff
-. Restart all master processes.
+. Restart all API servers.
```

**Comment by ahardin-rh** [comment]
@php-coder @smarterclayton This is updated. PTAL

**Review by php-coder — APPROVED** [review]
Thanks for your improvements!

**Comment by ahardin-rh** [comment]
Revision history in original PR

**Comment by adellape** [comment]
[Document: [security] [SCCFSI] Encrypt secrets at REST in etcd](https://trello.com/c/gY2hdWkD/559-document-security-sccfsi-encrypt-secrets-at-rest-in-etcd)

**Pull request merged**